### PR TITLE
Generalize software->licensed material in permission/condition/limitation descriptions

### LINKED ISSUE #743: The term "software" is used for cc-by-sa-4.0
The explanations for Permissions, Conditions and Limitations when the mouse hovers the items under each section use the word "software" whereas material should be used, here: https://choosealicense.com/licenses/cc-by-sa-4.0/

### PR DESCRIPTION
**Pull request by mlinksva**
Fixes #743

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -1,43 +1,43 @@
 permissions:
-- description: This software and derivatives may be used for commercial purposes.
+- description: This licensed material and derivatives may be used for commercial purposes.
```
Comment (waldyrious) [inline _data/rules.yml]: I'd say "licensed" is somewhat redundant (or rather, not very natural-sounding) when preceded by "this". How about omitting it in these cases, and leaving it for "the licensed material" only?

Comment (waldyrious) [reply _data/rules.yml]: Alternatively, replace "this" with "the", which would make these sentences better match the structure of the ones in the "conditions" section.

Comment (mlinksva) [reply _data/rules.yml]: Hmm, how about dropping "This"? It doesn't really make sense on choosealicense.com, and I'm not sure it helps when there is licensed software/material present either.

Comment (mlinksva) [reply _data/rules.yml]: I made the above and a couple other changes. How does it look to you now @waldyrious?

Comment (waldyrious) [reply _data/rules.yml]: Looks better, but personally I wouldn't say the "the"s were unhelpful. Without them the text feels more vague and legalese-like. Having the articles makes the text more concrete, and in a sense, more down-to-earth and approachable. Hope this makes sense.

Comment (mlinksva) [reply _data/rules.yml]: I prefer it, but yes makes sense -- I re-added the "the"s in e94c302

**Review by waldyrious — APPROVED** [review]
LGTM, FWIW!

**Pull request merged**